dist/svg.js

```diff
@@ -2414,15 +2415,28 @@ SVG.Matrix = SVG.invent({
     }
     // Skew
   , skew: function(x, y, cx, cy) {
-      return this.around(cx, cy, this.native().skewX(x || 0).skewY(y || 0))
+      // support uniformal skew
+      if (arguments.length == 1) {
+        y = x
+      } else if (arguments.length == 3) {
+        cy = cx
+        cx = y
+        y = x
+      }
+
+      // convert degrees to radians
+      x = SVG.utils.radians(x)
+      y = SVG.utils.radians(y)
+
+      return this.around(cx, cy, new SVG.Matrix(1, Math.tan(y), Math.tan(x), 1, 0, 0))
     }
     // SkewX
   , skewX: function(x, cx, cy) {
-      return this.around(cx, cy, this.native().skewX(x || 0))
```

Comment:
There was also an issue with line, it should have been something like this (was REALLY HARD to spot this one XD):
```javascript
return this.around(cx, cy, new SVG.Matrix().native().skewX(x || 0))
```
Since no other transform (scale, rotate, ...) use the native function, to fix it, I just made it call skew.